schedule/default_install_staging.yaml

```diff
@@ -0,0 +1,30 @@
+---
+name:           default_install
+description:    >
+  Test for default installation, with default options.
+conditional_schedule:
+   sle15accept_licence:
+    VERSION:
+      15-SP1:
+        - installation/accept_license
+schedule:
+  - installation/bootloader
+  - installation/welcome
+  - {{15-SP1accept_licence}}
```

Comment:
Seems that condition name doesn't match and I would use different naming.

Reply:
fixed on ba01a8a but i havent changed the name.